File: templates/Uwp/_composition/MVVMBasic/Feature.IdentityOptionalLogin/ViewModels/SettingsViewModel_postaction.cs
namespace Param_RootNamespace.ViewModels
{
    public class SettingsViewModel : Observable
    {
        public bool IsBusy
        {
            get { return _isBusy; }
            set
            {
                Set(ref _isBusy, value);
//^^
//{[{
                LogInCommand.OnCanExecuteChanged();
                LogOutCommand.OnCanExecuteChanged();
//}]}
            }
        }
    }
}

File: templates/Uwp/_comp/MVVMBasic/Service.IdentityOptionalLogin._VB/ViewModels/SettingsViewModel_postaction.vb
Namespace ViewModels
    Public Class SettingsViewModel
        Inherits Observable

        Public Property IsBusy As Boolean
            Get
                Return _isBusy
            End Get
            Set(value As Boolean)
                [Set](_isBusy, value)
'^^
'{[{
                LogInCommand.OnCanExecuteChanged()
                LogOutCommand.OnCanExecuteChanged()
'}]}
            End Set
        End Property
    End Class
End Namespace
